modules/nf-core/windowmasker/mk_counts/main.nf

```diff
@@ -0,0 +1,43 @@
+process WINDOWMASKER_MKCOUNTS {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::blast=2.13.0"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.13.0--hf3cf87c_0':
+        'quay.io/biocontainers/blast:2.13.0--hf3cf87c_0' }"
+
+    input:
+    tuple val(meta), path(ref)
+
+    output:
+    tuple val(meta), path("*.txt")  , emit: counts
+    path "versions.yml"             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args    = task.ext.args     ?: ""
+    def prefix  = task.ext.prefix   ?: "${meta.id}"
+
+    def memory = 3072
+    if (!task.memory) {
+        log.info '[WINDOWMASKER: MK_COUNTS] Available memory not known - defaulting to 3GB. Specify process memory requirements to change this.'
+    } else {
+        memory = (task.memory.toMega()*0.8).intValue()
```

Comment:
You can use the full memory here. The subsetting here is specifically for tools using Java :)

Reply:
Unless it makes some sense to limit here too (I don't really know this tool)

Reply:
It's a C tool, so you're right. No point in limiting it.
Done